### DISCUSSION TITLE
Geräteversatz manuell einstellen

### DISCUSSION THREAD

**Discussion by Martin9569** [Courseplay/Courseplay_FS22#1298] [Q&A] [answered]
Hallo, ist es in den neuen Versionen möglich den Gerätevesatz manuell einstellen?
Leider funktioniert bei mir, die Automatische ermittlung nicht so richtig. Beim rückwärts fahren fehlen immer ca. 1-2 Meter.
Ich nutze gemoddete Trecker, sowie gemoddete Geräte.

**Comment by Tensuko** [comment, marked as answer]
War es die ganze Zeit schon, beim Pflug wird es dynamisch berechnet und das muss auch so bleiben.
